docs/tutorials/remote_soc.rst

```diff
@@ -0,0 +1,209 @@
+Building Your Own SoC
+=====================
+
+This tutorial will walk you through the process of building an ASIC containing one PicoRV32 RISC-V CPU core and 2 kilobytes of SRAM, on an open-source 130nm Skywater process node.
+
+You can find complete example designs which reflect the contents of this tutorial in the public SiliconCompiler repository, `with RAM <https://github.com/siliconcompiler/siliconcompiler/tree/main/examples/picorv32_ram>`_ and `without RAM <https://github.com/siliconcompiler/siliconcompiler/tree/main/examples/picorv32>`_.
+
+See the :ref:`Installation <Installation>` section for information on how to install SiliconCompiler, and the :ref:`Remote Processing <Remote Processing>` section for instructions on setting up the remote workflow.
+
+[TODO / CR feedback: Add a screenshot here (or at the bottom)?]
+
+Download PicoRV32 Verilog Code
+------------------------------
+
+The heart of any digital design is its HDL code, typically written in a language such as Verilog or VHDL. High-level synthesis languages are gaining in popularity, but most of them still output their final design sources in a traditional HDL such as Verilog.
+
+PicoRV32 is an open-source implementation of a small RISC-V CPU core, the sort you might find in a low-power microcontroller. Its source code, license, and various tooling can be found `in its GitHub repository <https://github.com/YosysHQ/picorv32>`_.
+
+The repository contains many files, but the core CPU design is located in a single file called ``picorv32.v`` at the root of the repository.
+
+Create a new directory for this project, and copy the ``picorv32.v`` file into it.
+
+Because we are building this design as an ASIC rather than an FPGA bitstream, you will also need a constraints file to set a reference for the core clock signal. Create a file called ``picorv32.sdc`` in your new build directory, containing the following line::
+
+    create_clock -name clk -period 10 [get_ports {clk}]
+
+Build the PicoRV32 Core using SiliconCompiler
+---------------------------------------------
+
+Before we add the complexity of a RAM macro block, let's build the core design using the open-source Skywater130 PDK. Copy the following build script into the same directory which you copied ``picorv32.v`` into::
+
+    import siliconcompiler
+
+    chip = siliconcompiler.Chip('picorv32')
+    chip.load_target('skywater130_demo')
+    chip.input('picorv32.v')
```

Comment:
I think you're missing:
`chip.input('picorv32.sdc')`

Reply:
I'm planning to make a few changes which replace the SDC files with calls to `chip.clock()`

Reply:
Do we want this to be consistent throughout all the examples? In many other places right now, it's using `chip.input('<design>.sdc'.

Reply:
if the SDC is basically empty, then the .clock is probably better, otherwise we can use the SDC. The examples do not have to 100% consistent on this since we also need to show how to supply an SDC file